geoips/commandline/geoips_list.py

```diff
@@ -667,8 +665,11 @@ class GeoipsList(GeoipsCommand):
     """Top-Level List Command for listing off GeoIPS Artifacts."""
 
     name = "list"
-    command_classes = [
+    generated_classes = CommandClassFactory(
         GeoipsListSingleInterface,
+        interfaces.__all__,
+    ).classes
```

Comment:
If we make my other suggested update in a future issue, we would also need to change this.

Reply:
See #708

Reply:
Sounds good! I'll make sure to implement what you recommended after this PR and its dependents get merged in. Resolving for now.